base/refvalue.jl

```diff
@@ -8,6 +8,31 @@ mutable struct RefValue{T} <: Ref{T}
     RefValue{T}(x) where {T} = new(x)
 end
 RefValue(x::T) where {T} = RefValue{T}(x)
+"""
+    isassigned(ref::RefValue) -> Bool
+
+Test whether the given [`Ref`](@ref) has been associated with a value.
+This is always true for a [`Ref`](@ref) for a bitstype.
```

Comment:
(to avoid "for" repetition... feel free to ignore)
```suggestion
This is always true for a [`Ref`](@ref) of a bitstype object.
```